staking/validators/v1/nova_validators_dev.json

```diff
@@ -0,0 +1,37 @@
+{
+    "preferred":
+    {
+        "91b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c3":
+        [
+            "127zarPDhVzmCXVQ7Kfr1yyaa9wsMuJ74GJW9Q7ezHfQEgh6",
+            "13JuwkvSqGUDo8zErgfC9ivGfKfcdDyceFkvh9NW4wz7NbuF"
+        ],
+        "b0a8d493285c2df73290dfb7e61f870f17b41801197a149ca93654499ea3dafe":
+        [
+            "DhK6qU2U5kDWeJKvPRtmnWRs8ETUGZ9S9QmNmQFuzrNoKm4",
+            "EtETk1FbrDg7FoAfkREuXT7xHxCjbEf28sBvWf6zfB5wFyV",
+            "HYqFHkWAXTdYKYRU49VurZ5YnuT58gX7sVdL9weWi7XeczQ",
+            "Gb7iC1jnAJtSaNDoyvmr6aqbcthfhVMocCWiszbQynS6BDi",
+            "HuehfUJoctKux1gmMaN2d62SMSZEA3cjKfX2zmJkJEaRxwo",
+            "D991k2qmiWrVZzkehGP2QYSLdGH3s2gBvHZzpxQrzX4KSvn"
+        ],
+        "70255b4d28de0fc4e1a193d7e175ad1ccef431598211c55538f1018651a0344e":
+        [
+            "5DBhSX89qijHkzUt9gcqsq9RiXxDfbjxyma1z78JSCdt4SoU"
+        ],
+        "7eb9354488318e7549c722669dcbdcdc526f1fef1420e7944667212f3601fdbd":
+        [
+            "5A5Qgq3wn6JeH8Qtu7rakxULpBhtyqyX8iNj1XV8WFg3U58T"
+        ],
+        "b91746b45e0346cc2f815a520b9c6cb4d5c0902af848db0a80f85932d2e8276a":
+        [
+            "5DBhSX89qijHkzUt9gcqsq9RiXxDfbjxyma1z78JSCdt4SoU"
+        ]
+    },
+    "blacklisted": {
+      "b91746b45e0346cc2f815a520b9c6cb4d5c0902af848db0a80f85932d2e8276a":
+        [
+            "5FjdibsxmNFas5HWcT2i1AXbpfgiNfWqezzo88H2tskxWdt2"
```

Comment:
```suggestion
            "5DvV9ja7y1iLwyyHcXUhSTTPLBt6pfXzcjtEQBVDvLPMrCFc"
```

Reply:
![image](https://github.com/user-attachments/assets/8404b3ad-a57f-4863-b799-52c10914a8bb)
seems to be active atm